TESTING.md

```diff
@@ -0,0 +1,71 @@
+# Testing
+
+We have two different processes. One for running unit and integration tests and another one for functional tests. This is because functional tests usually take longer.
```

Comment:
```suggestion
We have two different processes. One for running unit and integration tests and another one for functional tests (also called end-to-end tests). This is because functional tests usually take longer.
```

> unit tests
> integration tests
> functional tests

Let's add some links or definitions to these concepts, e.g.:

https://en.wikipedia.org/wiki/Unit_testing
https://en.wikipedia.org/wiki/Integration_testing
https://en.wikipedia.org/wiki/Functional_testing

Maybe we could include a picture like this pyramid:
https://medium.com/@Colin_But/define-testing-strategy-using-the-testing-pyramid-1dabee37e823